api/src/main/java/org/motechproject/nms/api/web/contract/FrontLineWorkerUser.java

```diff
@@ -7,8 +7,8 @@
     private long currentUsageInPulses;
     private long endOfUsagePromptCounter;
```

Comment:
For incoming objects definitely.  For objects we send back, I think if the
field is required would we want to use the primitive so we can't assign
null to it?

On Fri, Apr 24, 2015 at 4:28 PM, kosh <notifications@github.com> wrote:

> In
> api/src/main/java/org/motechproject/nms/api/web/contract/FrontLineWorkerUser.java
> <https://github.com/koshalt/mim/pull/90#discussion_r29093691>:
>
> > @@ -7,8 +7,8 @@
> >      private long currentUsageInPulses;
> >      private long endOfUsagePromptCounter;
>
> did we standardize on the primitive type vs. object types (Long). I would
> think that we would use Long for contract serialization?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/koshalt/mim/pull/90/files#r29093691>.
>